### PR TITLE
Added real strings for legal approval & fixed order of share buttons to ...

### PR DESCRIPTION
**Pull request by bensternthal**
...match wireframe.

### CODE REVIEW

**Comment by bensternthal** [comment]
@sgarrity legal was asking to see the real copy to sign off on the legal bug.. so I went ahead and added. Note I also switched the order of the twitter/fb icons to match the wireframes.

File: mrburns/main/templates/choice-modal.html [hunk 1]
```diff
@@ -97,21 +97,21 @@
...
+                <span class="title">{{ _('Twitter') }}</span>
+              </button>
             </div>
           </div>
         </div>
       </div>
       <div class="choice-footnotes">
-        <div class="link"><a href="#" class="share-local">{{ _('Share on this site') }}</a></div>
-        <p class="note">{{ _('By sharing your choice with us, you can add your voice to our global map. We use geolocation to show you how people around the world share your vision for the Web.') }}</p>
+        <div class="link"><a href="#" class="share-local">{{ _('Share only on') }} webwewant.mozilla.org</a></div>
```
Comment (sgarrity) [inline mrburns/main/templates/choice-modal.html]: Will the URL be at the end of this phrase for all locales? Otherwise, we'll need a placeholder.

Comment (sgarrity) [inline mrburns/main/templates/choice-modal.html]: Beyond that, r+

**Comment by bensternthal** [comment]
@sgarrity yep you are correct and this is how i had it in the strings file. Should be good to go.

**Pull request merged**